eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -52,46 +52,16 @@ parameters:
       SubscriptionConfiguration: $(sub-config-gov-test-resources)
     China:
       SubscriptionConfiguration: $(sub-config-cn-test-resources)
-- name: Platforms
+- name: MatrixConfigs
   type: object
   default:
-    Linux:
-      Pool: azsdk-pool-mms-ubuntu-1804-general
-      OSVmImage: MMSUbuntu18.04
-      TestTargetFramework: netcoreapp2.1
-    Windows_NetCoreApp:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestTargetFramework: netcoreapp2.1
-      SupportsRecording: true
-    Windows_NetCoreApp_ProjectReferences:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestTargetFramework: netcoreapp2.1
-      AdditionalTestArguments: "/p:UseProjectReferenceToAzureClients=true"
-    Windows_NetFramework:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestTargetFramework: net461
-    Windows_NetFramework_ProjectReferences:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestTargetFramework: net461
-      AdditionalTestArguments: "/p:UseProjectReferenceToAzureClients=true"
-    MacOS:
-      Pool: Azure Pipelines
-      OSVmImage: "macOS-10.15"
-      TestTargetFramework: netcoreapp2.1
-      SupportedClouds: 'Public'
-    Windows_Net50:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestTargetFramework: net5.0
-    Windows_Net50_ProjectReferences:
-      Pool: azsdk-pool-mms-win-2019-general
-      OSVmImage: MMS2019
-      TestTargetFramework: net5.0
-      AdditionalTestArguments: "/p:UseProjectReferenceToAzureClients=true"
+    - Name: NET_live_test_base
+      Path: eng/pipelines/templates/stages/platform-matrix.json
+      Selection: all
```

Comment:
I suppose you want this to be sparse instead all.

Reply:
See my above comments re: all

Reply:
I'm not sure which comment you are referring to could you paste a direct link.

Reply:
Sorry, many more comments happened after this one :) This actually should be sparse now, because of the platform matrix changes you suggested.